lib/rollbar/plugins/rails/error_subscriber.rb

```diff
@@ -0,0 +1,9 @@
+module Rollbar
+  class ErrorSubscriber
+    def report(error, handled:, severity:, context:, source: nil)
+      extra = context.is_a?(Hash) ? context.deep_dup : {}
+      extra[:custom_data_method_context] = source
+      Rollbar.log(severity, error, extra)
+    end
```

Comment:
`handled:` is unused?

Reply:
Yes, though I'm considering using the `capture_uncaught` flag here to swallow the error when `!handled && !capture_uncaught`.